### PR TITLE
Surface asset check severity

### PR DESCRIPTION
**Pull request by johannkm**
Pipe severity up to gql

### CODE REVIEW

**Comment by johannkm** [comment]
Current dependencies on/for this PR:
* master
  * **PR #16139** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/16139" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/16139?utm_source=stack-comment).

File: python_modules/dagster-graphql/dagster_graphql/schema/asset_checks.py [hunk 1]
```diff
@@ -97,24 +102,21 @@ class GrapheneAssetCheck(graphene.ObjectType):
...
-    def __init__(self, name: str, asset_key: AssetKey, description: Optional[str] = None):
-        self._name = name
+    def __init__(self, asset_key: AssetKey, asset_check: ExternalAssetCheck):
+        self._asset_check = asset_check
         self._asset_key = asset_key
-        self._description = description
 
-    def resolve_name(self, _):
-        return self._name
+        self.name = asset_check.name
```
[the remark this answers is not in the document]
Comment (johannkm) [reply python_modules/dagster-graphql/dagster_graphql/schema/asset_checks.py]: We do it sometimes with graphene but yeah it's weird, removed

File: python_modules/dagster-graphql/dagster_graphql/schema/asset_checks.py [hunk 2]
```diff
@@ -97,24 +102,25 @@ class GrapheneAssetCheck(graphene.ObjectType):
...
+        return self._asset_check.asset_key
+
+    def resolve_name(self, _):
+        return self._asset_check.name
 
     def resolve_description(self, _):
-        return self._description
+        return self._asset_check.description
+
+    def resolve_severity(self, _):
```
Comment (sryza) [inline python_modules/dagster-graphql/dagster_graphql/schema/asset_checks.py:117]: Return type plz

**Review by sryza — APPROVED** [review]

**Comment by johannkm** [comment]
### Merge Activity

* **Aug 28, 2:59 PM**: @johannkm merged this pull request with [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/16139).

**Pull request merged**